styles/components/Logo.module.scss

```diff
@@ -0,0 +1,14 @@
+.link {
+  @apply w-1/2 h-auto mb-4;
+}
+
+.logoWrapper {
```

Comment:
Can we just make this something more like:
```scss
.link {
  @apply w-1/2 h-auto mb-4;
  div {
    @apply relative;
    @apply w-full h-full;
    @apply pt-[100%];
  }
  img { 
    @apply object-cover rounded-xl;
    @apply bg-black;
  }
}
```
This will help simplify the css a lot and make thigns a bit easier to read, it doesn't really negatively impact us to have less classes either. also maybe we change the `link` style to a `container` as its the container of the component.